internal/proto/local/controller/api/resources/accounts/v1/account.proto

```diff
@@ -43,8 +44,16 @@ message Account {
 	// The attributes that are applicable for the specific Account type.
 	oneof attrs {
 		google.protobuf.Struct attributes = 100 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.subtype) = "default"];
-		PasswordAccountAttributes password_account_attributes = 101 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.subtype) = "password"];
-		OidcAccountAttributes oidc_account_attributes = 102 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.subtype) = "oidc"];
+		PasswordAccountAttributes password_account_attributes = 101 [
+			(google.api.field_visibility).restriction = "INTERNAL",
+			(custom_options.v1.generate_sdk_option) = true,
+			(custom_options.v1.subtype) = "password"
```

Comment:
I added newlines between these field options, I'm happy to revert it back to a single line if we prefer.